gplately/tools.py

```diff
@@ -67,39 +137,80 @@ def extract_feature_lonlat(features):
     return rlon, rlat
 
 
-def lonlat2xyz(lon, lat):
+def lonlat2xyz(lon, lat, degrees=False):
```

Comment:
We should make a choice now to either stick with degrees or radians by default. Since we are earth scientists, not mathematicians, I propose we work in degrees hereon.